runners/google-cloud-dataflow-java/pom.xml

```diff
@@ -288,11 +288,21 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.avro</groupId>
```

Comment:
I'm surprised this validation required adding dependencies on avro and datastore. Is there a reason these dependencies appeared now but not before?